facts/Vagrantfile

```diff
@@ -177,6 +177,22 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     host.vm.provision 'shell', path: 'get_facts.sh'
     host.vm.provision 'shell', inline: '/sbin/shutdown -h now'
   end
+  config.vm.define 'oraclelinux-8-x86_64', autostart: false do |host|
+    host.vm.box = 'bento/oraclelinux-8'
+    host.vm.synced_folder '.', '/vagrant'
+    host.vm.provision 'file', source: 'Gemfile', destination: 'Gemfile'
+    host.vm.provision 'shell', inline: 'sysctl -w net.ipv6.conf.all.disable_ipv6=0'
```

Comment:
https://github.com/voxpupuli/facterdb/pull/340 was merged that makes this line redundant.